zerver/tests/test_user_groups.py

```diff
@@ -140,6 +139,17 @@ def test_user_group_update(self) -> None:
         result = self.client_patch('/json/user_groups/1111', info=params)
         self.assert_json_error(result, "Invalid user group")
 
+        self.logout()
```

Comment:
Add tests here as well as in other cases to make sure that administrators can modify the groups